src/components/UI/Card.tsx

```diff
@@ -1,8 +1,11 @@
 import "./Card.css";
-
-function Card(props) {
+type classProps = {
+  className: string;
+  children: JSX.Element[];
+};
+const Card: React.FC<classProps> = (props) => {
```

Comment:
If you are only adding `React` for the `FC` type, it's better to destructure it and only import `FC`. It will reduce the overall bundle size.